test/client/unit/specs/store/getters/user/userGems.js

```diff
@@ -1,13 +0,0 @@
-import { gems as userGems } from 'client/store/getters/user';
```

Comment:
why was this removed?

Reply:
this test was folded into the user.test.js file. I probably should have just renamed the file.